### ISSUE TITLE
ui_adaptor: accessibility regression

### ISSUE THREAD

**Issue by alex19EP** [CleverRaven/Cataclysm-DDA#39499]
# Describe the bug

when the interface migrates to ui_adapter, accessibility for the blind is impaired.
what is affected:
* crafting menu
* construction menu.

# Steps To Reproduce

1. use speakup on linux
2. press numpad star to enable tracking 
3. open construction or crafting menu
4. arrow between items.
5. item name isn't spoken

# Expected behavior

speakup announces item name. because it is Highlighted.

# Versions and configuration

* OS: archlinux
* Game Version: 217f4235ab
* Graphics version: terminal
* Ingame language: ru

# Additional context

i think if current item selection had been indicated not only by a color change, but also by the system cursor, this would remove this and many other problems.

**Comment by Qrox** [comment]
I'm not familliar with speakup so I'm not sure what change exactly caused it to not announce the item name. Also I don't have a linux environment to test it, so perhaps someone else could check what's causing the issue here?

**Comment by alex19EP** [comment]
@Qrox Honestly, I do not hope that anyone here knows this technology. but I'm sure that if we put the system cursor on the selected item, then the problem will be solved.
this seems like a simple solution to me, but I'm not big enough to do it myself yet.

**Comment by Qrox** [comment]
Should the cursor be at the beginning or the end of the item name?

**Comment by alex19EP** [comment]
> Should the cursor be at the beginning or the end of the item name?

at the beginning. thanks.
